app/src/actions/user.js

```diff
@@ -57,7 +57,7 @@ export function logout() {
       type: LOGOUT
     });
     window.location.hash = window.location.hash.replace(/#access_token=([a-zA-Z0-9.\-_]*)/g, '');
-    if (window.location.pathname.match('^\/map')) {
+    if (window.location.pathname.match('/map')) {
```

Comment:
The ^ caret is necessary, because it means: look for /map at the beginning of the path. The slash didn't need to be escaped.
I don't know why this error was not caught by my eslint.